app/controllers/concepts_controller.rb

```diff
@@ -84,10 +85,11 @@ def show_tree
       get_class(params) #application_controller
       
       not_found(t('concepts.missing_roots')) if @root.nil?
-
+      raw_ontology_uri_pattern = LinkedData::Client::HTTP.get("ontologies/#{@ontology.acronym}/latest_submission", {display: 'uriRegexPattern,preferredNamespaceUri'}) 
```

Comment:
don't use `.get` if a model exist, in this case it is `@ontology.latest_submission` or `@submission = get_ontology_submission_ready(@ontology)`

Reply:
@ontology.latest_submission is nil here, and the @submission = get_ontology_submission_ready(@ontology) will get all the submission and I need only these two fields uriRegexPattern,preferredNamespaceUri, is there a model to get a submission with the desired params?

Reply:
`@ontology.explore.latest_submission(include: 'uriRegexPattern,preferredNamespaceUri')`